packages/workflow/src/records/fhir.ts

```diff
@@ -774,7 +777,7 @@ export function createRegisterTask(previousTask: SavedTask): Task {
     'REGISTERED'
   )
 
-  const comments = previousTask?.note?.[0]?.text
+  const comments = comment ?? previousTask?.note?.[0]?.text
```

Comment:
Will this overwrite a comment made by the registrar while registering? @naftis

Reply:
Good point, and yes it will. This is fine though as per @jpye-finch comment. For events v2 there will be a more elegant audit log.